api/valor_api/schemas/filters.py

```diff
@@ -303,3 +305,28 @@ class Filter(BaseModel):
         extra="forbid",
         protected_namespaces=("protected_",),
     )
+
+
+def field_to_tuple(field_info):
+    if field_info.annotation in []:
+        pass
+
+
+# we want to pass a Filter as a query parameters instead of a body
+# so we make a new model `FilterQueryParams` where every value is a JSON string
+model_fields = Filter.model_fields
+model_def_dict = {kwarg: (str | None, None) for kwarg in model_fields}
+FilterQueryParams = create_model(
+    "FilterQueryParams",
+    __config__=ConfigDict(extra="forbid"),
+    **model_def_dict,  # type: ignore
+)
+
+
+def filter_query_params_to_filter(filter_query_params) -> Filter:
+    return Filter(
```

Comment:
can we add a docstring for consistency with other non-private functions in this file?

Reply:
yep, added